README.md

```diff
@@ -40,8 +40,7 @@ The following attributes are required for each user:
 * groups - A list of supplementary groups for the user.
 * append - If yes, will only add groups, not set them to just the list in groups (optional).
 * profile - A string block for setting custom shell profiles.
-* ssh_key - This should be a list of SSH keys for the user (optional). Each SSH key
-  should be included directly and should have no newlines.
+* ssh_key - This should be a list of SSH keys for the user (optional). Each SSH key should have no newlines. The source of ssh key can be the file system, github url e.g. http://github.com/username.keys or a string directly in the file.
```

Comment:
If this is a clarification, the authorized_key module actually supports urls not only github. This feature was added in ansible 1.9 and the current version of this role supports 1.3+ so this would also need to note that or bump the minimum version to 1.9.